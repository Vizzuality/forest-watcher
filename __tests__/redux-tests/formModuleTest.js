import 'react-native';
import formReducer from 'redux-modules/form';

describe('Redux Form Module', () => {
  it('Initial reducer state', () => {
    expect(formReducer(undefined, { type: 'NONE' })).toMatchSnapshot();
  });
});
