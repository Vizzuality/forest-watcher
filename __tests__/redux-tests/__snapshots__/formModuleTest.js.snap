// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Redux Form Module Initial reducer state 1`] = `
Object {
  "migrated": true,
  "oldData": Object {},
}
`;
